jenkins/Pipeline.gy

```diff
@@ -4,7 +4,7 @@ pipeline {
         stage('Linting & Unit Tests') {
             steps {
                 echo 'Linting and legacy unit tests'
-                sh 'jenkins/run jenkins/tox -r --current-env -e jenkins-unittests,py3-server'
+                sh 'jenkins/run jenkins/tox -r --current-env -e jenkins-unittests'
```

Comment:
If we're going to do a Pbench Server 0.69 update, shouldn't we have unit testing for it?

Reply:
Yes, we do, see `jenkins-unittests`.